### ISSUE TITLE
Compilation errors in version 1.1.0

### ISSUE THREAD

**Issue by avra911** [Nosto/nosto-magento2#43]
Hello,

In the latest version `1.1.0` we receive a lots of errors during `bin/magento setup:di:compile`, here is one sample output:

```
	Nosto\Tagging\Helper\Account
		Incorrect dependency in class Nosto\Tagging\Helper\Account in /var/www/magento/vendor/nosto/module-nostotagging/Helper/Account.php
\Magento\Framework\Module\Manager already exists in context object
Total Errors Count: 1
```

They are relatively easy to fix, however they break automatic deploys with production mode.
Magento version is `2.1.3` if is required to know.

**Comment by hannunosto** [comment]
Thanks for reporting this @avra911. We will look into this in near future.

**Comment by hannunosto** [comment]
Fixed in 1.1.1
